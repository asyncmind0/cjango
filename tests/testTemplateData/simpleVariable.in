My first name is {{ first_name }}. My last name is {{ last_name }}.
