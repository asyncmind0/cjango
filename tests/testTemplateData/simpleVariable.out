My first name is John. My last name is Smith.
